### PR TITLE
LPS-126286

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by LocalizedSimpleFieldsetsStructure#CheckFieldsetTranslationsOnStructure

**Comment by emanuelfisilva** [comment]
Thanks for your review @Tim-Cao , but we'll choose to keep these validations separated.

File: portal-web/test/functional/com/liferay/portalweb/macros/PortletEntry.macro [hunk 1]
```diff
@@ -33,6 +33,13 @@ definition {
 		}
 	}
 
+	macro changeLocaleInModal {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/PortletEntry.macro:36]: Use existing macro

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path [hunk 2]
```diff
@@ -598,6 +598,11 @@
 	<td>//div[contains(@id,'rotate')]/a[contains(@class,'btn-link')][2]</td>
 	<td></td>
 </tr>
+<tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:601]: Use existing locator

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Modal.path [hunk 3]
```diff
@@ -33,6 +33,11 @@
 	<td>//div[contains(@class, 'modal-footer')]//button[contains(@class,'btn-primary')][contains(text(),'Save')]</td>
 	<td></td>
 </tr>
+<tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Modal.path:36]: Use existing locator

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path [hunk 4]
```diff
@@ -54,6 +54,11 @@
 	<td>//div[contains(@class,'dropdown-menu') and contains(@class,'show')]//button[contains(@class,'dropdown-item') and contains(.,'${key_locale}')]</td>
 	<td></td>
 </tr>
+<tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:57]: Use existing locator

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsetsStructure.testcase [hunk 5]
```diff
@@ -259,12 +259,64 @@ definition {
...
+        WebContentStructures.addCP(structureName = "WC structure name");
+
+        DataEngine.addFieldsets(
+            createNewFieldset = "true",
+            fieldNameList = "Text",
+            fieldsetLabel = "${fieldsetLabel}",
+            fieldsetName = "${fieldsetLabel}"
+        );
+
+        DataEngine.gotoFieldsetItemEllipsis(fieldsetName = "${fieldsetLabel}");
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsetsStructure.testcase]: Use existing macro to simplify test

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3784
View total diff https://github.com/liferay-echo/liferay-portal/pull/3784/commits/60ee6b892558c03476187bb3cbe86bb2d7abd3a7

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**